table/table.go

```diff
@@ -647,6 +647,9 @@ func (t *Table) blockOffsetsCacheKey() []byte {
 	return buf
 }
 
+// IndexSize size is the length of block indices
+func (t *Table) IndexSize() int { return len(t.blockIndex) }
```

Comment:
len(t.blockIndex) isn't the actual size of the table index. The actual size can be found as
```go
	indexSz := 0

	for _, bi := range t.blockOffsets() {
		indexSz += bi.Size()
	}
	return indexSz
```

Use the `blockOffsets()` function and not `t.blockIndex`. `t.blockIndex` can be nil if indices are cached.

Reply:
Thanks. Done.